src/components/web-form-renderer.vue

```diff
@@ -305,36 +405,18 @@ const uploadAttachment = async (attachment, instanceId) => {
  */
 const handleSubmit = async (payload, callback) => {
   if (props.actionType === 'preview') {
-    showModal(submissionModal, { type: 'previewModal' });
-  } else {
-    const { data: [data], status } = payload;
-    showModal(sendingDataModal);
-    if (status !== 'ready') {
-      // Status is not ready when Form is not valid and in that case submit button will be disabled,
-      // hence this branch should never execute.
-      return;
-    }
-
-    const instanceId = await postPrimaryInstance(data.instanceFile);
-
-    if (instanceId) {
-      const attachmentRequests = data.attachments.map(a => () => uploadAttachment(a));
-      const attachmentResults = await runSequentially(attachmentRequests);
-
-      // TODO: what to do if attachments upload fail - blocked, need to define requirements / UX
-      if (attachmentResults.every(r => !isProblem(r))) {
-        callback({ next: POST_SUBMIT__NEW_INSTANCE }); // tell OWF to clear out the Form
-        if (isPublicLink.value) {
-          showModal(submissionModal, { type: 'thankYouModal' });
-          formVersionXml.reset(); // hides the Form
```

Comment:
Is this line still important? Or was it not necessary before?

Reply:
I wanted to keep this line but it is making styling of modal weird. 

Previously `thankYouModal` was hideable so it was important to hide the Form but now it is not